dependency-check-suppressions.xml

```diff
@@ -43,4 +43,8 @@
     <suppress>
         <cve>CVE-2020-5398</cve>
     </suppress>
+
```

Comment:
try not to have any cve

Reply:
Please add latest nimbus jar, and remove CVE suppression 17195 , refer commit  https://github.com/hmcts/ccpay-bulkscanning-app/pull/195/files#diff-c197962302397baf3a4cc36463dce5eaR222